### PR TITLE
Update listVault.json

### PR DESCRIPTION
**Pull request by rahulalapati43**
## ARM API Information (Control Plane)

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow.
</i>
Azure 1st Party Service can try out the [Shift Left](https://aka.ms/ShiftLeft) experience to initiate API design review from ADO code repo.  If you are interested, may request engineering support by filling in with the form https://aka.ms/ShiftLeftSupportForm.

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. By default, Azure SDKs of all languages (.NET/Python/Java/JavaScript for both management-plane SDK and data-plane SDK, Go for management-plane SDK only ) MUST be refreshed with/after swagger of new version is published. If you prefer NOT to refresh any specific SDK language upon swagger updates in the current PR, please leave details with justification here.

### Contribution checklist (MS Employees Only):
- [ ] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

> **Applicability**: :warning:
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs
> - Adding new properties
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them.

### Breaking Change Review Checklist
If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by adarce — APPROVED** [review]

File: specification/keyvault/resource-manager/readme.md [hunk 1]
```diff
@@ -223,6 +223,13 @@ directive:
     - R4015 # The 'MHSMPrivateEndpointConnection' sub-resource doesn't define List as per Network Team's specification.
     - R2005 # The 'ManagedHsms_PurgeDeleted' operation should not return a mix of 202 and syncronous return types (200, 201, 204) as directed by SDK team.
     - R4009 # Vault object is the only one that need to be tracked with SystemData
+- suppress:
```
Comment (raych1) [inline specification/keyvault/resource-manager/readme.md:226]: You have the `api-version` defined inline of get operation but validation uses the common definition at 
https://github.com/Azure/azure-rest-api-specs/blob/ece224eae9f30e64c5e9f28991f45ea4c937bc7a/specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2022-02-01-preview/keyvault.json#L2062

To bypass this is to ignore `where` and one suppression can only include one error code. 

So, it should be like this:
```yml
- suppress: INVALID_REQUEST_PARAMETER
   from: 
   reason: 
- suppress: OBJECT_MISSING_REQUIRED_PROPERTY
   from: xx
   where: 
      - '$.paths./subscriptions/{subscriptionId}/resources.get.parameters[2]'
   reason:
```

Comment (rahulalapati43) [reply specification/keyvault/resource-manager/readme.md]: The suggested suppression directives didn't work. Even the simple suppression directives without any where and from conditions are not working: 
``` yaml
directive:
- suppress:
    - R3016 # The 'release_policy' property for KeyCreateParameters does not support camelCase.
    - R3026 # The 'PrivateEndpointConnection' and 'PrivateLinkResource' sub-resources don't define PATCH as per Network Team's specification.
    - R3025 # The 'PrivateLinkResource' is only accessible via List operation; does not define GET as per Network Team's specification.
    - R4015 # The 'MHSMPrivateEndpointConnection' sub-resource doesn't define List as per Network Team's specification.
    - R2005 # The 'ManagedHsms_PurgeDeleted' operation should not return a mix of 202 and syncronous return types (200, 201, 204) as directed by SDK team.
    - R4009 # Vault object is the only one that need to be tracked with SystemData
    - OBJECT_MISSING_REQUIRED_PROPERTY
    - INVALID_REQUEST_PARAMETER
Could you please take a look and let me know.

File: specification/keyvault/resource-manager/readme.md [hunk 2]
```diff
@@ -218,11 +218,24 @@ input-file:
...
     - R2005 # The 'ManagedHsms_PurgeDeleted' operation should not return a mix of 202 and syncronous return types (200, 201, 204) as directed by SDK team.
+- suppress:
     - R4009 # Vault object is the only one that need to be tracked with SystemData
+- suppress: INVALID_REQUEST_PARAMETER
+  from: keyvault.json
+  reason: The Vaults_List API endpoint only supports version 2015-11-01.
+- suppress: OBJECT_MISSING_REQUIRED_PROPERTY
+  from: keyvault.json
+  where:
```
Comment (raych1) [inline specification/keyvault/resource-manager/readme.md:236]: Can you remove `where` for now as this api-version error isn't supported by where suppression?

File: specification/keyvault/resource-manager/readme.md [hunk 3]
```diff
@@ -218,11 +218,24 @@ input-file:
...
     - R3026 # The 'PrivateEndpointConnection' and 'PrivateLinkResource' sub-resources don't define PATCH as per Network Team's specification.
+- suppress:
     - R3025 # The 'PrivateLinkResource' is only accessible via List operation; does not define GET as per Network Team's specification.
+- suppress:
     - R4015 # The 'MHSMPrivateEndpointConnection' sub-resource doesn't define List as per Network Team's specification.
+- suppress:
     - R2005 # The 'ManagedHsms_PurgeDeleted' operation should not return a mix of 202 and syncronous return types (200, 201, 204) as directed by SDK team.
+- suppress:
     - R4009 # Vault object is the only one that need to be tracked with SystemData
+- suppress: INVALID_REQUEST_PARAMETER
```
Comment (raych1) [inline specification/keyvault/resource-manager/readme.md:231]: The yaml title has typo. It should be:

### Supression

File: specification/keyvault/resource-manager/readme.md (hunk 2 above)
Comment (rahulalapati43) [reply specification/keyvault/resource-manager/readme.md]: Removed the where here: https://github.com/Azure/azure-rest-api-specs/pull/22155/commits/eb0e90c24fe3ba5f8b29c94a0a5a407a8797c504. The validation is failing: https://github.com/Azure/azure-rest-api-specs/pull/22155/checks?check_run_id=10665304132

File: specification/keyvault/resource-manager/readme.md (hunk 3 above)
Comment (rahulalapati43) [reply specification/keyvault/resource-manager/readme.md:226]: You mean, update suppress to supression?

**Pull request merged**